consensus/istanbul/protocol.go

```diff
@@ -22,25 +22,21 @@ import (
 
 // Constants to match up protocol versions and messages
 const (
+	// No longer supported, can be removed.
+	// The corresponding version upstream (eth/65) is removed in upstream PR #22636.
 	Celo64 = 64 // eth/63 + the istanbul messages
```

Comment:
Do we want to remove these now or wait for the upstream PR?

Reply:
My inclination is to leave them for now, but I wouldn't be opposed to removing them either.

Reply:
sounds good